src/model/url-collection-mode.ts

```diff
@@ -0,0 +1,6 @@
+export type UrlCollectionMode = 'full' | 'no_page'
+
+export const UrlCollectionModes: Record<string, UrlCollectionMode> = {
```

Comment:
Does autocompletion work for the fields of this? If not, we can just make it a plain record

Reply:
I think it does, didn't think about it, but I believe used it a couple times